### PR TITLE
Show warning message when wrong credentials passed

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

**Comment by Joozty** [comment]
@viktor-yakubiv Not sure if we want to have some close button there or make message disappear after some time.

File: public/login.html [hunk 1]
```diff
@@ -80,3 +81,18 @@ <h1>Login</h1>
...
+<script>
+  function login(e) {
+    e.target.action = 'https://api.dev.core.ac.uk/login_check?continue=' + encodeURIComponent(
+      window.location.origin
+    );
+  }
+
+  const urlParams = new URLSearchParams(window.location.search);
+  if (urlParams.has('reason')) {
+    const message = document.getElementById('error-message')
```
Comment (viktor-yakubiv) [inline public/login.html]: I am considering using more old JS but whatever works. I am not sure about our browsers list yet. We actually support only latest Chrome but we will need to wide this up.

Maybe `var` and avoiding `URLSearchParams` in favour of pure regular expression test would work better.

File: public/login.html [hunk 2]
```diff
@@ -80,3 +81,18 @@ <h1>Login</h1>
...
+    message.innerHTML='The username or password you entered is incorrect.';
+    message.style.display = 'block';
```
Comment (viktor-yakubiv) [inline public/login.html]: Simple CSS snippet may help you avoid this.

```css
.message:empty { display: none }
```

Comment (Joozty) [reply public/login.html]: This is really cool selector. Thanks

**Pull request merged**